department-of-veterans-affairs/abd-vro
# Resolves a Snyk Gate Check SecRel failure for Inadequate Encryption Strength

https://github.com/department-of-veterans-affairs/abd-vro-internal/security/code-scanning/1650

Snyk is simply asking us to specify the TLS version we are instantiating in this line. It is a simple enough change to follow the Snyk recommendation, so even though it doesn't seem necessary, there is also no need to fight it.

## What was the problem?
SecRel is failing due to a Snyk scan that calls out the potential ambiguity of instantiating "TLS" vs "TLSv1.2"

Associated tickets or Slack threads:
- #1579 

## How does this fix it?[^1]
It follows the recommendation from Snyk, which is to explicitly call out "TLSv1.2" when instantiating.

## How to test this PR
- Rerun SecRel and confirm that the Snyk Gate Check is no longer failing


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
